src/clients/inst_root.rb

```diff
@@ -168,18 +162,8 @@ def main
         )
       )
 
-      if @check_CA_constraints
-        @helptext = Ops.add(
-          @helptext,
-          Builtins.sformat(
-            # additional help text about password
-            _(
-              "<p>If you intend to use this password for creating certificates,\nit has to be at least %1 characters long.</p>"
-            ),
-            @pw_min_CA
-          )
-        )
-      end
+      validator = Users::CAPasswordValidator.new
+      @helptext << validator.help_text if validator.enabled?
```

Comment:
It is probably just POV, as for me helptext provide info how to use functionality and if it is not enabled, then it return empty string, but maybe in future we decide to add something like `"Validating password for CA is not enabled. If you want to use it, do this and this"` so it is more like helptext for provided functionality. So not help for what is *can* provide, instead help for what it *really* provides.